10up/ElasticPress
# Fix: autosuggest breaks search blocks

### Description of the Change
Autosuggest changes the frontend markup of the search field to add the suggestion list, while it works for the normal search field, it's breaking the search block styling due to CSS conflict between autosuggest and search block. Because the markup of the search block is predictable, this PR adds a separated behavior for search field in the search blocks that reuses the block markup to add the suggestion list and required class.


### Alternate Designs
n/a


### Benefits
Make autosuggest work with search block, which is a preparation for full site editing.


### Possible Drawbacks
n/a


### Verification Process
1. Enable and configured Autosuggest features.
2. Create a new page and add a search block.
3. View the page on the front end, type a keyword, see the suggestions.


### Checklist:






### Applicable Issues
#2211


### Changelog Entry
Fixed the styling issue of Autosuggest and search block.
